src/main/java/org/jenkinsci/plugins/configfiles/maven/job/MvnGlobalSettingsProvider.java

```diff
@@ -131,13 +134,20 @@ public String getDisplayName() {
             return "provided global settings.xml";
         }
 
-        public ListBoxModel doFillSettingsConfigIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project) {
-            project.checkPermission(Item.CONFIGURE);
-            
+        public ListBoxModel doFillSettingsConfigIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project, String currentValue) {
```

Comment:
Isn't `currentValue` missing a `@QueryParameter`?

Reply:
changed and the name as well